data/.schema.yaml

```diff
@@ -153,6 +159,8 @@ properties:
                 $ref: "#/$defs/negative_positive_number"
               analyte:
                 type: string
+              limit_of_quantification:
```

Comment:
Let's include `unknown` as an option just for consistency with the loq in the `analytes` section. Something like this?

```yaml
limit_of_quantification:
  description: Lowest value at which the analyte can be reliably quantified.
  oneOf:
    - type: number
    - const: unknown
```

Reply:
Good point!

Reply:
Lesson learned! Thanks for the explanation. Appreciate the guidance!